cmd/migrate/migrate.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/ledgerwatch/bolt"
+	"github.com/ledgerwatch/turbo-geth/common/dbutils"
+)
+
+func encodeToCBOR(buf []byte) ([]byte, error) {
+	acc, err := decodeRLP(buf)
+
+	if err != nil {
+		return nil, err
+	}
+
+	res := make([]byte, acc.EncodingLengthForStorage())
+	acc.EncodeForStorage(res)
+	return res, nil
+}
+
+// This method does not Commit
+func convertDatabaseToCBOR(db *bolt.DB, maxTxOperations uint) error {
+	var k, v []byte
+	var done bool = false
+	for !done {
+		var i uint
+		err := db.Update(func(tx *bolt.Tx) error {
+			var accountBucket *bolt.Bucket = tx.Bucket(dbutils.AccountsBucket)
+			c := accountBucket.Cursor()
+
+			for k, v = c.Seek(k); k != nil; k, v = c.Next() {
+
+				enc, err := encodeToCBOR(v)
+				if err != nil {
+					return err
+				}
+				i++
+				err = accountBucket.Put(k, enc)
+				if err != nil {
+					return err
+				}
+				if i+1 == maxTxOperations {
+					break
+				}
+			}
+
+			if k == nil {
+				done = true
+			}
+			return nil
```

Comment:
Please make a copy of what is in `k` here, because it will become invalid at the end of transaction:
````
k = common.CopyBytes(k)
````